lesson5_HT3/task3_2.js

```diff
@@ -0,0 +1,14 @@
+const number = 5
+let i = 1
+while (i <= 10){
+    console.log(`5 x ${i} = ${number*i}`)
```

Comment:
If you change the `number` variable value you should change also string. You can avoid it using interpolation 
```js
console.log(`${number} x ${i} = ${number*i}`)
```